docs/Contributors_Guide/continuous_integration.rst

```diff
@@ -119,12 +380,31 @@ Here is a list of the currently supported keywords and what they control:
 
 * **ci-skip-all**: Don't run anything - skip all automation jobs
 * **ci-skip-use-cases**: Don't run any use cases
+* **ci-skip-unit-tests**: Don't run the Pytest unit tests
 * **ci-run-all-cases**: Run all use cases
 * **ci-run-diff**: Obtain truth data and run diffing logic for
   use cases that are marked to run
 * **ci-run-all-diff**: Obtain truth data and run diffing logic for
   all use cases
-* **ci-only-docs**: Only run build documentation job - skip the rest
+
+.. _cg-ci-get-image:
+
+Create/Update METplus Docker Image
+----------------------------------
+
+This job calls the **docker_setup.sh** script (found in .github/jobs).
+This script builds a METplus Docker image and pushes it to DockerHub.
+The image is pulled instead of built in each test job to save execution time.
+The script attempts to pull the appropriate Docker image from DockerHub
+(dtcenter/metplus-dev:**<BRANCH_NAME>**) if it already exists so that unchanged
+components of the Docker image do not need to be rebuilt.
+
+DockerHub Credentials
+^^^^^^^^^^^^^^^^^^^^^
+
+The credentials needed to push images to DockerHub are stored in Secret
+Environment Variables for the repository. These variables are passed
```

Comment:
Are these repo-specific secrets or are they now organization-wide secrets? Maybe this detail doesn't matter.